core/src/main/java/tc/oc/pgm/wool/MonumentWool.java

```diff
@@ -50,7 +50,7 @@ public MonumentWool(MonumentWoolFactory definition, Match match) {
   public String toString() {
     StringBuilder sb = new StringBuilder("MonumentWool{");
     sb.append("id=").append(this.getId());
-    sb.append(",owner=").append(this.owner.getName());
+    sb.append(",owner=").append(this.owner.getDisplayName());
```

Comment:
If this is the only reason we keep `getDisplayName`, then just remove it from `toString`.